tikv-client/src/main/java/com/pingcap/tikv/expression/visitor/RangePartitionLocator.java

```diff
@@ -59,6 +66,7 @@ public Boolean visit(ComparisonBinaryExpression node, PartitionLocatorContext co
         Expression expression = left.getChildren().get(0);
         ColumnRef columnRef = (ColumnRef) expression;
         columnRef.resolve(tableInfo);
+        dataType = IntegerType.BIGINT;
```

Comment:
Why set YEAR as IntergerType? Shall we add an annotation here?

Reply:
function year() return integer